### ISSUE TITLE
ADD A Dedicated Page for Github Contributors 

### ISSUE THREAD

**Issue by shubhagarwal1** [ajaynegi45/Uttarakhand-Culture-NewUI#198]
refrence design 
<img width="491" alt="Screenshot 2024-10-25 at 4 11 35 AM" src="https://github.com/user-attachments/assets/3b90676d-2b59-4128-b323-9fa4b650621d">

**Comment by ajaynegi45** [comment]
> refrence design <img alt="Screenshot 2024-10-25 at 4 11 35 AM" width="491" src="https://private-user-images.githubusercontent.com/105449260/379955814-3b90676d-2b59-4128-b323-9fa4b650621d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk4MzA4OTQsIm5iZiI6MTcyOTgzMDU5NCwicGF0aCI6Ii8xMDU0NDkyNjAvMzc5OTU1ODE0LTNiOTA2NzZkLTJiNTktNDEyOC1iMzIzLTlmYTRiNjUwNjIxZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyNVQwNDI5NTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02OGRiNDg1OGZmYzAwNDc4MTEzMmFhMjAyNjNjYTFhZTk0YmM3MjViNjNhMDEwOGYwMDdlYjczODhiZDZlYjE1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Tzj7SsSisvH0bmU_C9CLTpCkNvLwPVeoAceo3CR590g">

Hi @shubhagarwal1,

Thank you for expressing your interest in working on this issue. I'm delighted to inform you that I have assigned this issue to you. Your willingness to contribute to our project is much appreciated.

Match the design with current design. ref https://ukculture.netlify.app/contributors

Before making any contributions, please read the Readme .md file carefully to better understand the project's goals and purpose. This will give you clarity on our mission. Also, make sure to review the Contributing .md file, which contains important guidelines to make the contribution process smoother, especially for newcomers.

Feel free to start working, and if you have any questions or need assistance during the process, please don't hesitate to reach out.

**Comment by yashksaini-coder** [comment]
@shubhagarwal1 bhai can I work on this

**Comment by shubhagarwal1** [comment]
> @ajaynegi45 @shubhagarwal1 bhai can I work on the contributors page??

Yes i have no problem, with it, i have prepared some boiler plate code, i will share that with you.

**Comment by ajaynegi45** [comment]
> > @ajaynegi45 @shubhagarwal1 bhai can I work on the contributors page??
> 
> Yes i have no problem with it, i have prepared some boiler plate code, i will share that with you.

Hi @shubhagarwal1 @yashksaini-coder,

Hope you're doing great! 😊 Just wanted to check in and see how things are coming along with the issue you're working on. Any progress or updates you’d like to share? And if you’ve run into any roadblocks or need a hand with anything, please don’t hesitate to reach out!

Looking forward to hearing from you!

**Comment by shubhagarwal1** [comment]
Hey @ajaynegi45, once confirmed if yash is not working on the issue, I can present a draft to you.

**Comment by yashksaini-coder** [comment]
> Hey @ajaynegi45, once confirmed if yash is not working on the issue, I can present a draft to you.

Go ahead @shubhagarwal1 bhaiji you can work on it. I am preparing for my clg exams

**Comment by ajaynegi45** [comment]
> Hey @ajaynegi45, once confirmed if yash is not working on the issue, I can present a draft to you.

Ask him
